src/Xamarin.Forms.PancakeView.Multi/Xamarin.Forms.PancakeView.Multi.csproj

```diff
@@ -99,7 +100,7 @@
         <PackageReference Include="SkiaSharp.Views" Version="1.68.3" />
     </ItemGroup>
     <ItemGroup Condition="'$(TargetFramework)' == 'monoandroid10.0'">
-      <PackageReference Include="Xamarin.AndroidX.AppCompat.AppCompatResources" Version="1.2.0.2" />
+        <PackageReference Include="Xamarin.AndroidX.AppCompat.AppCompatResources" Version="1.2.0.2" />
```

Comment:
I'm out of the loop on `Xamarin.AndroidX.AppCompat.AppCompatResources`.

How come it is necessary?

For example, here is a File -> New Xamarin.Forms project where I've only added `Xamarin.Forms.PancakeView` v2.1.0.714 + `Xamarin.Android.Support.v7.AppCompat` v 28.0.0.3, and it compiles without any build errors.

<img width="831" alt="Screen Shot 2020-09-28 at 8 47 58 AM" src="https://user-images.githubusercontent.com/13558917/94454967-686a7a80-0167-11eb-952f-1df900b5a43a.png">

Reply:
I'm also very much out of the loop of all this Android magic happening lately. This came in through #130 so perhaps @saamerm could comment. I'm also wondering whether or not to undo the changes on AndroidX done through that PR. Afaik AndroidX and Android10 are not the same thing and fixing Android 10 compatibility is probably an easier job than doing the AndroidX migration atm.

Reply:
Yeah, I definitely believe that it's a tough one. It's why I've held off on it so long. I'm very grateful some brave people have ventured into this though. I'm just trying to figure out what the best way moving forward is to get out of this Android 10 / Android X hole :)

Reply:
> You were able to build it, and were you able to run the sample successfully as well?

Yes, the sample runs without errors

> I don't remember why EXACTLY I needed the AppCompatResources 

Are you able to pin down why `Xamarin.AndroidX.AppCompat.AppCompatResources` is required? 

I'm hesitant to add too many NuGet packages as dependencies because adding unnecessary NuGet packages (and I'm not saying that `Xamarin.AndroidX.AppCompat.AppCompatResources` is unnecessary) can cause dependency conflicts with existing projects.

Reply:
Hi Friends! What do you think about this plan to move forward? 

For now, let's add a dependency for only `Xamarin.Android.Support.v7.AppCompat`, since it has been documented that adding `Xamarin.Android.Support.v7.AppCompat` solves a reported bug: https://github.com/sthewissen/Xamarin.Forms.PancakeView/issues/135#issuecomment-698537745.

If a user in the future hits a different bug that requires `Xamarin.AndroidX.AppCompat.AppCompatResources`, then we can add it as a second dependency.